movieapi/mainapp/tasks.py

```diff
@@ -0,0 +1,24 @@
+from celery import Celery
+from celery import shared_task
+from django.core.mail import send_mail
+from django.core.exceptions import BadRequest, ValidationError, RequestAborted
+
+app = Celery("tasks", broker="pyamqp://guest@localhost//")
```

Comment:
```suggestion
app = Celery("tasks", broker="pyamqp://guest@localhost//")
```
ovo bi izvuka u .env BROKER_URL="pyamqp://guest@localhost//"